README.md

```diff
@@ -9,10 +9,29 @@
 For monitoring metrics of a RavenDB instance (not its data) use the RavenDB Telegraf plugin.  
 Please refer to: [Monitoring RavenDB Metrics](https://ravendb.net/docs/article-page/latest/http/server/administration/monitoring).
 
-All query examples bellow are based on RavenDB's [sample data](https://ravendb.net/docs/article-page/latest/csharp/studio/database/tasks/create-sample-data).
- 
+**In this readme**:
+* [Sample dashboard showing data from RavenDB](#sample-dashboard-showing-data-from-ravendb)
+* [Install the RavenDB data source plugin](#install-the-ravendb-data-source-plugin)
+* [Set RavenDB as your data source - Unsecure server](#set-ravendb-as-your-data-source---unsecure-server)
+* [Set RavenDB as your data source - Secure server](#set-ravendb-as-your-data-source---secure-server)
+* [Querying Features](#querying-features)
+
+# Sample dashboard showing data from RavenDB
+
 ![Screen](https://github.com/ravendb/ravendb-grafana-datasource/raw/main/src/img/dashboard_screen.png)
 
+# Install the RavenDB data source plugin
+
+1. Download latest Grafana from [here](https://grafana.com/grafana/download).
+2. Download the latest RavenDB data source plugin zip file (**ravendb-datasource-1.0.0.zip**) from [here](https://github.com/ravendb/ravendb-grafana-datasource/releases).
```

Comment:
maybe 1.x.x? To avoid hardcoding version?

Reply:
done